app/javascript/dashboard/components/widgets/conversation/ConversationBulkActions.vue

```diff
@@ -0,0 +1,280 @@
+                <thumbnail
+                  src="agent.thumbnail"
+                  :username="agent.name"
+                  size="22px"
+                  class="margin-right-small"
+                />
+                <span class="reports-option__title">{{ agent.name }}</span>
+              </div>
+            </li>
+          </ul>
+          <div v-else class="agent-confirmation-container">
+            <p>
+              {{
+                $t('BULK_ACTION.ASSIGN_CONFIRMATION_LABEL', {
+                  conversationCount: conversations.length,
+                })
+              }}
+              <strong>
+                {{ selectedAgent.name }}
+              </strong>
+            </p>
+            <div class="agent-confirmation-actions">
+              <woot-button
+                color-scheme="primary"
+                variant="smooth"
+                @click="goBack"
+              >
+                {{ $t('BULK_ACTION.GO_BACK_LABEL') }}
+              </woot-button>
+              <woot-button
+                color-scheme="primary"
+                variant="flat"
+                :is-loading="uiFlags.isUpdating"
+                @click="submit"
+              >
+                {{ $t('BULK_ACTION.ASSIGN_LABEL') }}
+              </woot-button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import Thumbnail from 'dashboard/components/widgets/Thumbnail.vue';
+import { mapGetters } from 'vuex';
+
+export default {
+  components: {
+    Thumbnail,
+  },
+  props: {
+    conversations: {
+      type: Array,
+      default: () => [],
+    },
+    allConversationsSelected: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      showAgentsList: false,
+      query: '',
+      selectedAgent: null,
+    };
+  },
+  computed: {
+    ...mapGetters({
+      agents: 'agents/getAgents',
+      uiFlags: 'bulkActions/getUIFlags',
+    }),
+    filteredAgents() {
+      return this.agents.filter(agent =>
+        agent.name.toLowerCase().includes(this.query.toLowerCase())
+      );
+    },
+  },
+  mounted() {
+    this.$refs.selectAllCheck.indeterminate = true;
+  },
+  methods: {
+    assignAgent(agent) {
+      this.selectedAgent = agent;
+    },
+    selectAll(e) {
+      this.$emit('selectAllConversations', e.target.checked);
+    },
+    goBack() {
+      this.selectedAgent = null;
+    },
+    submit() {
+      this.$emit('assignAgent', this.selectedAgent);
+    },
+    resolveConversations() {
+      this.$emit('resolveConversations');
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.flex-between {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.bulk-action__container {
+  padding: var(--space-normal) var(--space-one);
+  background-color: var(--s-50);
+  position: relative;
+  box-shadow: 6px 3px 22px 9px rgb(181 181 181 / 50%);
```

Comment:
Can you reduce the color or increase the blur radius or remove the shadow from the right side? Looks darker.

<img width="650" alt="image" src="https://user-images.githubusercontent.com/64252451/167768421-3b59bac9-f1f3-4e73-b52e-4d735146c1ea.png">